lib/shellcheck/analysis.bash

```diff
@@ -12,6 +12,7 @@ analyze() {
         "--check-sourced"
         "--norc"
         "--format=json1"
+        "--exclude=SC2190" # Temporarily disable: https://github.com/koalaman/shellcheck/issues/2393
```

Comment:
SC2190 is valuable feedback in many situations. It can be incorrect in one specific situation.
It's good to not show incorrect feedback in that specific (albeit, IMO, not very common) situation.
But it's also good to flag all the more common situations.

Given how common the various situations are, I'm leaning towards keeping SC2190.